.travis.yml

```diff
@@ -60,6 +60,10 @@ install:
       fi
     fi
 
+  # Perceptual image hashing (TBD: push recipe to conda-forge!)
+  - conda install pip
```

Comment:
pip is installed by default when you create a new conda environment (e.g. https://travis-ci.org/SciTools/iris/jobs/170017955#L253) so it doesn't need to be done separately. The result of this line is a no-op (https://travis-ci.org/SciTools/iris/jobs/170017955#L464).